src/main/java/com/github/mkopylec/charon/core/mappings/ConfigurationMappingsProvider.java

```diff
@@ -24,6 +24,7 @@ protected boolean shouldUpdateMappings(HttpServletRequest request) {
     protected List<Mapping> retrieveMappings() {
         return charon.getMappings().stream()
                 .map(Mapping::copy)
+                .sorted((o1, o2) -> o2.getPath().compareTo(o1.getPath()))
```

Comment:
I think the sorting should be done in `MappingsCorrector` class:

```java
    public void correct(List<Mapping> mappings) {
        if (isNotEmpty(mappings)) {
            mappings.forEach(this::correctMapping);
            int numberOfPaths = mappings.stream()
                    .map(Mapping::getPath)
                    .collect(toSet())
                    .size();
            if (numberOfPaths < mappings.size()) {
                throw new CharonException("Duplicated destination paths in mappings");
            }
// return sorted mappings
        }
    }
```

Now every custom `MappingsProvider` will have the mappings sorted, not only the default one.